docs/GetStarted/createProfileACM.md

```diff
@@ -1,6 +1,10 @@
 --8<-- "References/abbreviations.md"
 
-Admin Control Mode (ACM) provides full access to Intel® Active Management Technology (Intel® AMT) functionality. User consent is optional for redirection features.
+Admin Control Mode (ACM) provides full access to Intel® Active Management Technology (Intel® AMT) functionality. User consent is optional for supported redirection features:
+
+- **IDE Redirection:** Share and mount images remotely with a specified storage media (e.g., USB flash drive).
```

Comment:
We should put a disclaimer on the IDER bullet that while AMT supports it, the toolkit doesn't natively right now.  Same on CCM page